### PR TITLE
Chart component for new dashboard...

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description of change
This is the first stage of adding a chart component into the new dashboard that will reflect how many investment projects you have. 

Note: Don't care too much about styling for now as we are not sure if this will resemble the final component. The styling I have added is loosely based on the wireframe until we know more from user research.

## Test instructions
- Run storybook
- Run the new dashboard, see #3248 for instructions

## Screenshots
Screenshot_2021-01-20_at_10 46 47

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

### CODE REVIEW

File: src/client/components/InvestmentProjectSummary/index.jsx [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+        param: 2,
+        value: 20,
+      },
+      {
+        key: 'Active',
+        param: 3,
+        value: 5,
+      },
+      {
+        key: 'Verfiy win',
```
Comment (elcct) [inline src/client/components/InvestmentProjectSummary/index.jsx:35]: Typo

Comment (paulgain) [reply src/client/components/InvestmentProjectSummary/index.jsx:35]: Looks good.
 
**Storybook**
Can we add another financial year to the chart's data so we can see the component update when toggling the years.

**Comment by paulgain** [comment]
Looks good.

**Storybook**
Can we add another financial year to the chart's data so we can see the component update when toggling the years?

**Comment by ian-leggett** [comment]
> Looks good.
> 
> **Storybook**
> Can we add another financial year to the chart's data so we can see the component update when toggling the years?

I agree I should have added another option. I have added this now and seems to work nicely, let me know your thoughts 👍

**Review by elcct — APPROVED** [review]

**Review by paulgain — APPROVED** [review]

**Pull request merged**